### DISCUSSION TITLE
KSP 2 Support

### DISCUSSION THREAD

**Discussion by Henr1k-H1** [ebkr/r2modmanPlus#1229] [Ideas]
Would you consider adding support for Kerbal Space Program 2?

**Comment by MythicManiac** [comment]
Hey, currently the proper forum for game requests is on the Thunderstore Discord here: https://discord.thunderstore.io/, you can find a forum channel titled "game requests" there.

As far as KSP2 goes, I believe it was requested already and has an unlisted community on the website here: https://thunderstore.io/c/kerbal-space-program-2/ but it didn't have a lot of traction from modders at the time at least.
